packages/dev/core/src/ObjectModel/objectModelInterfaces.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * A container with an original object and an accessor that allows modifying properties
+ * on some other object.
+ */
+export interface IObjectAccessorContainer<T> {
+    object: any;
+    accessor: T;
+}

```

Comment:
It might be better to use a more generic naming for this interface so that the glTF one doesn't feel as awkward. Maybe something like this? Comment would require updating also.

```suggestion
/**
 * A container with an original object and an accessor that allows modifying properties
 * on some other object.
 */
export interface IObjectInfo<T> {
    object: any;
    info: T;
}
```